win32/readdir.c

```diff
@@ -1,3 +1,5 @@
+#define _CRTDBG_MAP_ALLOC
```

Comment:
Why is this needed?

Reply:
Could we avoid this by including the `"php.h"` header first?

Reply:
yes, that would also work

Reply:
I'd prefer doing that then, as the define isn't directly related to the code in this file.

Reply:
Okay, done. :)